demos/preemption_fairness/README.md

```diff
@@ -256,6 +283,8 @@ spec:
 
 Higher-priority workloads preempt lower-priority ones.
 
+<!-- Generated by 'make update-readme'. DO NOT EDIT. -->
+<!-- YAML-START: demos/preemption_fairness/resources/workloadpriority.yaml -->
```

Comment:
When running the Script the workload priorities dont get updated. 
They should be split up into 2 code blocks and given the same treatment as the other split files.

Reply:
There was a bug found, fixed in the latest commit. Thanks :)